pwndbg/commands/__init__.py

```diff
@@ -276,6 +277,9 @@ def _OnlyWithResolvedHeapSyms(*a, **kw):
         e = lambda s: print(message.error(s))
         w = lambda s: print(message.warn(s))
         if pwndbg.heap.current.can_be_resolved():
+            # We don't catch the error for developers, only catch for users.
+            if gdb.parameter("exception-verbose") or gdb.parameter("exception-debugger"):
```

Comment:
We should:
* fetch those through pwndbg.config.exception_verbose and pwndbg.config.exception_debugger
* move this to the exception case and re-raise the err `except Exception as err: if ...: raise err`

Reply:
Ok, I'll fix it with your suggestions!